### DISCUSSION TITLE
ZXSpectrum Emulator ?

### DISCUSSION THREAD

**Discussion by fulvioalessio** [fdivitto/FabGL#181] [Ideas]
Hi Fabrizio,

are you planning to implement a ZXSpectum emulator ?
There are some emulators, but I found problems (spiffs/etc) with lilygo vga32 device.
Instead, your library/examples works fine!

Good work! Complimenti per l'ottimo lavoro!
f

**Comment by fdivitto** [comment]
I'm going to add some emulators, including the ZX Spectrum. Unfortunately I don't know when I will actually be able to do it!

Grazie!
